addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py

```diff
@@ -60,6 +60,18 @@ def gather_material(blender_material, export_settings):
         pbr_metallic_roughness=__gather_pbr_metallic_roughness(blender_material, orm_texture, export_settings)
     )
 
+    print(material.pbr_metallic_roughness)
+    print("->", orm_texture)
```

Comment:
Did you intend to leave these `print` statements in here?  There's another one on line 70.

Reply:
Fixed